### PR TITLE
CT-913 Switch to using appflow-apps for wizard

### PR DESCRIPTION
**Pull request by jlane9**

### CODE REVIEW

**Review by johnwheeler-prod — APPROVED** [review]
definitely wait on jared's approval, but it looks good to me.

**Review by jaredcbaum — APPROVED** [review]

**Review by devinshoemaker — APPROVED** [review]

**Pull request merged**
